CONTRIBUTING.md

```diff
@@ -306,6 +307,6 @@ When adding support for a new Python release remember to:
 
 - Add the version in `tox.ini`
 - Add the version in `pyproject.toml` trove classifiers
-- Update github workflows accordingly; lint and benchmarks use the latest supported version
+- Update github workflows accordingly with `tox -e generate-workflows`; lint and benchmarks use the latest supported version
```

Comment:
This is valid for the `Bumping the Python baseline` chapter too

Reply:
Added in [b3e6e84](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/2977/commits/b3e6e84e596dafb32c13255cb5b7da12a8a56e87)